uc/uc.py

```diff
@@ -20,6 +20,7 @@ def marshalled(self, context):
         company_report = body[0]
         company_report.set('ns1:product', self.product_code)
         company_report.set('ns1:version', '2.1')
+        print(context.envelope.getChild('Body'))
```

Comment:
Thanks for the info @anah, but my concern with logging without creating a handler pointing to stdout is if heroku will collect it... do you know the answer to that?

I learned that things sent to stdout are collect into `heroku logs` [from here](https://devcenter.heroku.com/articles/python-faq#what-happens-to-data-written-to-standard-out).

But in their [log docs](https://devcenter.heroku.com/articles/logging#writing-to-your-log) the only example I see is pointing to stdout.